packages/cpf-generator/src/constants.js

```diff
@@ -0,0 +1,33 @@
+export const CPF_LENGTH_WITHOUT_MASK = 11;
+
+export const CPF_LENGTH_WITH_MASK = 14;
```

Comment:
I think that is better handle just CPF without a mask (you can achieve this using 'helper-only-numbers' package)

Reply:
thats true, and will make the function syntax easier.

Reply:
actually i has a point, why dont handle only with cpf with the mask then the people who want only the numbers uses the helper

Reply:
@lucassveloso was working on cpf mask, maybe with this feature we can return just a valid cpf number and the user choose if use a mask or not